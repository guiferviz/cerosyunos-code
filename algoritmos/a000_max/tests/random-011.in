17
30
1594308278 224118530 851399736 505752191 1701729216 902648589 447826744 2090232317 178478552 278562337 1552115871 2094869433 1890922226 1811079266 1519759927 1708441233 1924816605 2008915310 1239173985 1826138108 149395292 576087843 1273604096 1063381465 1890359859 605534092 801166735 1535936295 1754786160 1574974378
30
217464991 2111439291 31480748 1886980992 222866807 1930011458 513772200 2029430159 1803418728 627365520 293393470 752122512 858553853 1651548911 316679613 857803705 1049370349 1435462643 90180144 1295409546 1268082232 823625283 27176070 1560699768 1056211328 1403636483 88868610 1648486218 1772877554 1684579712
30
1102751904 972427909 789964633 1659712912 22723364 925636899 939971615 348495051 1311012776 1381984142 388459334 221453032 178249958 831711408 728564010 1225996935 1687942834 815990423 1719352929 2040543556 162843720 586072286 306035016 1368901992 394739498 2003614824 117341645 357603396 748176041 1804421173
30
1876537176 1064833172 1205259331 1274587781 918804307 1447960850 1960897271 1417318165 648732834 1466917174 1124658485 993722428 1120349930 981420423 869910319 1487092797 250693750 1752051908 500603748 1698113119 877828692 2142487334 509715451 861969432 773192016 918654390 676323731 1435731612 1260175647 1582801023
30
845238311 1570495637 632121494 1732083266 1778143104 40506469 2142762126 2041669273 2128781338 1793431889 486397757 1575622805 1300547698 1757233445 233793638 533055840 1184542517 1326657020 798811046 1248626270 291860167 1322260096 829889681 66079595 41925042 422616707 1784642173 1856579458 1637195028 1820598198
30
1494350267 996898741 1010959462 317144394 2079892068 1174980887 1403433243 913278075 1565059919 1493301968 871810340 2094033940 784768900 706331437 1975201449 824556475 444497068 1031898326 232796208 300001092 561491492 472894563 1997481520 537902051 832716816 823881195 835093490 1033387267 239094292 1221590731
30
1169882669 1544916548 487809496 1016926196 998570808 292404540 1012867636 231376177 1527903320 163645841 862676035 1878015635 2055535546 649681195 424690454 1715695174 1023005066 1838590924 1088269141 1468288303 256429216 1170337331 1674303769 1922954289 1462628964 665809597 172483089 1859272402 149031318 897421519
30
460349087 159071581 1365380907 1552572689 496706316 56788001 1809784583 902791082 1155672208 582264579 1043038585 964007158 375982489 1131467581 960044366 1552338101 447104103 1003763674 1429940531 119664655 879250718 5851869 492943501 654396987 567766888 1476077187 1654154066 2024705391 1903353471 1358075772
30
914500238 911027537 1853363216 1229620254 1789425719 1311639809 1410892661 1128129801 340624933 542205578 1655768822 772623402 1917837206 21974900 1125369260 67012768 255670597 1890647574 1290883942 232242567 968331257 1873236280 1853700904 111598404 384715965 172021590 1137515225 766500700 1435230173 531886218
30
1784613061 1252734537 708194423 744517680 551433929 914793019 873751309 1981278818 504429265 1453670612 1088310338 1576033409 195731345 1775795015 1282765007 9863035 1022677058 836338500 1920861109 1640572975 864419662 639196194 140644729 684290673 1187912575 1191763994 682939948 558808067 781845871 516849800
30
2067312560 1317502667 1059767311 2044480826 598476480 1940228940 1508470773 1497849087 2111236501 1046347718 1195528315 206322754 1746231361 723812848 1648184525 1923654555 169306901 387716501 1116084255 1273214542 755470432 1016559389 396283304 619014411 742173938 561602582 539789217 1353362414 916051426 940142488
30
631634004 1164138244 998275737 1205254968 650005958 427240686 1869596118 734489874 220158906 2019993942 1383010944 475767543 568225502 1579034116 1013090476 1463400753 781227298 624618297 822342011 1052313308 786526197 1239877572 184118827 27398068 1838575066 1163337 2015626752 480790460 13965759 1471290127
30
612470190 1582226247 369851777 426800729 2085358655 1386285639 1957680938 277753232 1870743905 189544513 38853118 1111470902 1107063504 2023418096 1362325689 123889357 1159209502 372910346 455457866 1417380415 1359879681 731988437 1485191417 1545055406 1119791088 1222566497 974424540 1613086587 581380640 548765808
30
299365000 521875763 1193190538 340074702 2054784362 112476543 584100585 1345937806 1983374996 182288225 573935055 90592179 820663385 648229170 597431988 335937000 786752899 1221824521 2063882681 2078734572 1818236039 644015113 1302571397 1095681351 50617668 749832814 1196128420 1664954724 1156631904 1278728804
30
1561749636 1790087811 1053782997 1523324011 2115901088 1651052729 499145093 111072251 865073024 1220312748 1030793916 430775847 357972877 68914947 1831402172 1947948934 1376236120 1247322094 21059479 644834985 115257733 955741359 200319797 1170496116 1851070230 793006428 223954720 1260055068 784799015 646798782
30
2057510279 1361554337 122280249 1563501611 455806244 1806501385 380763995 1040887139 2085498122 36795559 1232877187 1174520581 189653565 344824666 2025334378 848508969 67542002 766253070 1129321570 1895163234 1600541225 1082732779 569966544 302121500 538352022 125312704 355595353 570107774 29512777 1464567411
30
954454803 996703084 1625509999 249696178 541457440 1777245513 1831760442 635322831 1649811383 1942223237 1895682241 1867199057 2133881650 1961959232 1710360445 560504042 2040564359 1763234063 2004949143 1893896093 1178685384 1225898997 1130613561 888389462 2022910562 867501158 1838415362 1519931964 125805274 831066248
