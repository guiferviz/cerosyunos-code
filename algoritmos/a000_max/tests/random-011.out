2094869433
2111439291
2040543556
2142487334
2142762126
2094033940
2055535546
2024705391
1917837206
1981278818
2111236501
2019993942
2085358655
2078734572
2115901088
2085498122
2133881650
